Makefile

```diff
@@ -22,7 +22,7 @@ $(GO):
 # Run unit tests
 # TODO
 test: $(GO)
-	echo hello world
+	$(GO) test ./pkg/... -coverprofile cover.out
```

Comment:
Please add cover.out to .gitignore in this PR, as it belongs to this PR

Reply:
done